src/bigtiffimage.cpp

```diff
@@ -110,14 +110,16 @@ namespace Exiv2
                 byte buffer[8];
                 io.read(buffer, 2);
                 const int size = getUShort(buffer, byteOrder);
-                assert(size == 8);
 
-                io.read(buffer, 2); // null
+                if (size == 8)
```

Comment:
Maybe we should throw an exception when size != 8?

Reply:
done